reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -109,26 +110,25 @@ def run_cassandra(job):
     cutoff = job.sp.r_cut * u.nm
 
     seedslist = [
-        [7860904, 98601355],
-        [955793508, 417823039],
-        [254420642, 2130720464],
-        [58120272, 1465850411],
-        [757616664, 1940492980],
```

Comment:
Why did the seeds have to change?  I suppose it doesn't matter, as long as you use the same seedslist for each job (so that it's reproducible).  Did you use the new seedslist for all jobs?

Reply:
I was getting an error from MoSDeF Cassandra regarding the length of the seeds [here](https://github.com/MaginnGroup/mosdef_cassandra/blob/cc0d3605a3b6f7a0c952324fb92a5aeb91032dfd/mosdef_cassandra/writers/inp_functions.py#L778).

Reply:
Ah, that's probably due to the overflow that occurs in Cassandra if a seed goes over 2,147,483,648.  I made sure they didn't exceed that, but MoSDeF Cassandra won't let them exceed 1,000,000,000.